### DISCUSSION TITLE
What's the best way to change user information for sites only hosted on the local webserver?

### DISCUSSION THREAD

**Discussion by nevillepark** [datenstrom/community#931] [Ask a question] [answered]
Hi,

I have a Yellow site which isn't online—I run it locally (with `php yellow.php serve`) to use the web interface. I generate a static site and upload that to a static web host. 

I want to change the email associated with my username. 

Entering `php yellow.php user change [email password]` just seems to _switch_ the current user, not change any details.

In `yellow-user.ini` it says "Pending: <new email>:<hash>." I assume it tried to send an email but since there's no mail server set up, it failed? 

[This comment](https://github.com/datenstrom/yellow/discussions/865#discussioncomment-5766902) suggests there is a way to do this by just editing yellow-user.ini and entering a new bcrypt hash. But it doesn't say what is used to generate the hash, though I assume it's something like "username email password". What text _exactly_ do I need to enter for the hash? 
Thanks in advance!

**edit:** Surprisingly, just changing the email address in the "Email: " field in yellow-user.ini worked!

**Comment by wunderfeyd** [comment, marked as answer]
In your case, I would recommend (due to the unknown flags set) to simply remove the account with

`php yellow.php user remove [old email]`

and to set it up again

`php yellow.php user add [new email] [password]`.

Edit the details afterwards in your `yellow-user.ini`.

If the account is valid (not pending, active etc.) you could also change the `Email` line with a text editor in your `yellow-user.ini`.

**Reply by nevillepark** [reply]
Thanks so much!
